server/main.lua

```diff
@@ -343,3 +64,79 @@ ESX.RegisterServerCallback('esx_mechanicjob:getPlayerInventory', function(source
 
 	cb({items = items})
 end)
+
+--Start of Vehicle Code
+ESX.RegisterServerCallback('esx_mechanicjob:buyJobVehicle', function(source, cb, vehicleProps, type)
```

Comment:
which function calls this callback?

Reply:
Removd all "vehicle" codes (imported from esx_police, function call for esx_mechanicjob:buyJobVehicle missing from the original file).